gusb/gusb-device.c

```diff
@@ -956,6 +959,32 @@ g_usb_device_get_string_descriptor_bytes (GUsbDevice  *device,
 	return g_bytes_new (buf, rc);
 }
 
+/**
+ * g_usb_device_get_string_descriptor_bytes:
+ * @desc_index: the index for the string descriptor to retrieve
+ * @langid: the language ID
+ * @error: a #GError, or %NULL
+ *
+ * Get a raw string descriptor from the device. The returned string should be freed
+ * with g_bytes_unref() when no longer needed.
+ *
```

Comment:
maybe add something like "The descriptor will be at most 128 bytes in length, but you can use g_usb_device_get_string_descriptor_bytes_full() if either a smaller or larger descriptor should be requested." -- or something like that with better English.